submissions/oluwabusolamiEluyela/html/index.html

```diff
@@ -0,0 +1,81 @@
+<!doctype html>
+<html>
+  <head>
+<meta charset "UTF-8">
+  <title>justify.com</title>
+  </head>
+
+  <body>
+    <header>
+      <div> 
+       <h1><a href="#">JUSTIFY</a></h1>
+      </div>
+	 <h2><a href="#">Hot Deals</a>
+         <a href="#">Gift Cards</a>
+         <a href="#">Sell</a><c/enter>
+         <a href="#">Book Rentals</a></h2>
+	<ol>
+         <li><a href="#">New Releases</a></li>
+	 <li><a href="#">General Book Shelf</a></li>
+	 <li><a href="#">Today's Deals</a></li>
+         <li><a href="#">Registration</a></li> 
+	 <li><a href="#">Teachers Corner</a></li>
+	 <li><a href="#">Students Corner</a></li>
+         <li><a href="#">Contact</a></li>
+	 <li><a href="#">Order Offline</a></li>
+	</ol>
+  </header>
+   <main>
+    <div>
+     <li>
```

Comment:
You are using `<li>` tag without `<ul>` or `<ol>` tag. This is wrong. And it isn't used properly in this context

Reply:
Reviewed. Please check

Reply:
Welldone @Oluwabusiolami But I don't think ol and li tag is useful in this scenario or usage. I will advise that you remove them